### PR TITLE
Css canvas medical - Nimrod Castano

### PR DESCRIPTION
**Pull request by nimcastano**
Screenshot 2023-04-05 at 11 43 32 AM

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
Please include screenshots of the other sections, or a gif of you scrolling the whole page, so I can verify the whole page looks like the example image

**Comment by nimcastano** [comment]
css-canvas-medical gif

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -10,53 +10,101 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <div class="row" id="toptop">
-    <div class="col-half" id="canvas-logo">
-      <img id="logo" src="images/logo-blue.svg">
+  <div class="container">
+    <div class="row" id="toptop">
+      <div class="col-half" id="canvas-logo">
+        <img id="logo" src="images/logo-blue.svg">
```
Comment (shawnkost) [inline css-canvas-medical/index.html:16]: Don't forget the `alt` attribute

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -10,53 +10,101 @@
...
-  </div>
-  <div class="row" id="locations">
-    <div class="col-half">
-      <h2>Cascade Family Practice, Portland OR</h2>
-      <h2>Temescar Creek Medicine, Oakland CA</h2>
-      <h2>San Juan Healthcare, Friday Harbor WA</h2>
+    <div class="row" id="links-block">
+      <div class="address-col">
+        <div class="white-logo-container">
+          <img src="images/logo-white.png" class="white-logo">
```
Comment (shawnkost) [inline css-canvas-medical/index.html:81]: Don't forget the `alt` attribute

**Review by shawnkost — CHANGES_REQUESTED** [review]
Very nicely done Nimrod! Just a couple changes requested here

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request closed without merging**